### DISCUSSION TITLE
Is copilot down?

### DISCUSSION THREAD

**Discussion by jlyr** [community/community#13217] [Copilot Conversations]
This is the error message I'm receiving!
Please help!

[INFO] [ghostText] [2022-03-22T05:48:42.660Z] Breaking, no choices
[INFO] [default] [2022-03-22T05:48:42.795Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-03-22T05:48:43.541Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 746 ms
[INFO] [streamChoices] [2022-03-22T05:48:43.543Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4oRfnKc35JcQxxsEK3f7o8BfXeH79}] created: [{1647928123}]

**Comment by davecheney** [comment]
This looks normal to me. Are you receiving no completions at all? What language are you writing in? Which vscode version? Which extension version? Have you upgraded?

**Reply by jlyr** [reply]
The only suggestions I'm receiving is local:
[INFO] [ghostText] [2022-03-22T08:52:52.276Z] Found inline suggestions locally

I'm writing in both PHP and python. Suggestions are not being fetched for both with prompts that previously copilot could suggest working results.

VSC Details
Version: 1.65.2 (Universal)
Commit: c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1
Date: 2022-03-10T14:33:49.188Z (1 wk ago)
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin arm64 20.6.0

Yes, Im using Copilot v1.9.5350

**Reply by davecheney** [reply]
I’m sorry you’re having problems. What does the icon in the bottom right of the screen saying?

**Reply by jlyr** [reply]
![image](https://user-images.githubusercontent.com/80109128/159445541-5dcfdbad-da65-49e5-88e8-d17b55c02e60.png)

**Reply by jlyr** [reply]
<img width="1102" alt="Screenshot 2022-03-22 at 5 09 05 PM" src="https://user-images.githubusercontent.com/80109128/159445822-216d84db-1417-4f6c-887f-6361a7079d0e.png">

**Reply by davecheney** [reply]
It looks like copilot is installed and working, it just has nothing to offer for your particular circumstance at the moment.

**Reply by jlyr** [reply]
In previous versions, the suggestions could complete the code with the same prompt. I tried to re-install previous versions but the suggestion still remains status quo.

**Reply by davecheney** [reply]
From everything that I can see copilot is working correctly. Are you able to try a different project or language?

**Reply by liornabat** [reply]
I have the same issue. the language is Go.
No suggestions now.

**Comment by darker008** [comment]
I have the same issue. the language is javascript
I use sample code referring to the guide, but still no suggestion.
any one could hep?

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by n-r-w** [comment]
same issue
`
[INFO] [auth] [2023-02-15T19:23:25.381Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-02-15T19:23:25.382Z] GitHub Copilot could not connect to server. Extension activation failed: "Contact Support"
`

**Reply by DimaGitHahahab** [reply]
Are you located in Russia? Maybe there are some new sanctions from github lol

**Reply by megastruktur** [reply]
Same here, location: Belarus.
It works under the VPN though  :(

**Comment by n-r-w** [comment]
yes :(
